### ISSUE TITLE
drpqual not populated properly in drp_all.fits

### ISSUE THREAD

**Issue by kslong** [sdss/lvmdrp#124]
In version 1.0.3, the drpall-1.0.3.fits file has a keyword drpqual, which I assume should indicate whether the drp processes the exposure properly, but all of the values in this column are 0, including exposures for which there was no flux calibration.  If this keyword is supposed to catch problems, then this looks like an error.  If this keyword is not intended to catch this, then there ought to be a keyword that does.

**Comment by havok2063** [comment]
This is not a drpall file problem.  The drpall file pulls the DRPQUAL value from the headers of the lvmSFrame files.  We currently are not tracking any pipeline quality issues during the reductions, and are not updating this value.  That's why it is always 0, which is the default value we start at.  The pipeline has no quality assessment tracking at the moment.

**Comment by kslong** [comment]
Agreed. I have changed the name.  The real issue is that the DRPQUAL value is not populated in the DRP

**Comment by ajmejia** [comment]
This issue (the lack of quality flags for different reduction steps) is in the list we've been discussing the last weeks. I'll address this when it moves up in the list of priorities. I don't think this will be a stopper to make progress on other opened issues.
